cmd/cluster/azure/create.go

```diff
@@ -116,3 +125,32 @@ func applyPlatformSpecificsValues(ctx context.Context, exampleOptions *apifixtur
 	}
 	return nil
 }
+
+// lookupRHCOSImage looks up a release image and extracts the RHCOS VHD image based on the nodepool arch
+func lookupRHCOSImage(ctx context.Context, arch string, image string, pullSecretFile string) (string, error) {
```

Comment:
can we please extend TestDeserializeImageMetadata to validate this value is actually findable or cover the same with a test for lookupRHCOSImage?

Reply:
Yeah no problem. I pushed another commit to extend TestDeserializeImageMetadata.